src/actions/account.js

```diff
@@ -126,6 +132,59 @@ export const { initializeRecoveryMethod, validateSecurityCode, setupRecoveryMess
         wallet.validateSecurityCode.bind(wallet),
         () => defaultCodesFor('account.validateSecurityCode')
     ],
+    INIT_TWO_FACTOR: [
+        wallet.twoFactor.initTwoFactor.bind(wallet.twoFactor),
+        () => defaultCodesFor('account.initTwoFactor')
+    ],
+    REINIT_TWO_FACTOR: [
+        wallet.twoFactor.reInitTwoFactor.bind(wallet.twoFactor),
+        () => defaultCodesFor('account.reInitTwoFactor')
+    ],
+    SEND_TWO_FACTOR: [
+        wallet.twoFactor.sendRequest.bind(wallet.twoFactor),
+        () => defaultCodesFor('account.sendTwoFactor')
+    ],
+    RESEND_TWO_FACTOR: [
+        wallet.twoFactor.resend.bind(wallet.twoFactor),
+        () => defaultCodesFor('account.resendTwoFactor')
+    ],
+    VERIFY_TWO_FACTOR: [
+        wallet.twoFactor.verifyTwoFactor.bind(wallet.twoFactor),
+        () => defaultCodesFor('account.verifyTwoFactor')
+    ],
+    PROMPT_TWO_FACTOR: [
+        (requestPending = null) => {
```

Comment:
nit: why need `requestPending = null`? Isn't it easier to just check for `if (requestPending) {` later?

Reply:
removed